templates/careers/application/faq.html

```diff
@@ -45,7 +47,7 @@ <h2 class="p-faq-stepped-list__title">Employment</h2>
                 <button type="button" class="p-accordion__tab" id="employment-tab4" aria-controls="employment-tab4-section" aria-expanded="false"><span class="u-default-max-width">What is the travel expectation?</span></button>
               </div>
               <section class="p-accordion__panel" id="employment-tab4-section" aria-hidden="true" aria-labelledby="employment-tab4">
-                <p>In general we expect our people to be able to travel 2-4 times a year to attend our Sprints. Our Sprints are in-person events where our functions come together to plan roadmaps, share learnings, collaborate and work on initiatives&hellip; and simply have great fun!  The locations for these sprints vary but are usually between EMEA and AMER and they usually last 1 week, occasionally 2 depending on your involvement. For some roles i.e. Sales, Field, Alliances etc there will be a more regular requirement to travel. Please make sure that you ask about the specific requirements during your interview with the team.</p>
+                <p>In general we expect our people to be able to travel 2-4 times a year to attend our Sprints. Our Sprints are in-person events where our functions come together to plan roadmaps, share learnings, collaborate and work on initiatives&hellip; and simply have great fun!  The locations for these sprints vary but are usually between EMEA and AMER and they usually last 1 week, occasionally 2 depending on your involvement. For some roles i.e. Sales, Field, Alliances etc there will be a more regular requirement to travel. Please make sure that you ask about the specific requirements during your interview with the team.  If you have any restrictions on travel please, please let us know.</p>
```

Comment:
Sorry, I just noticed that there is also an extra space. Feel free to disregard this and proceed with merging though.
```suggestion
                <p>In general we expect our people to be able to travel 2-4 times a year to attend our Sprints. Our Sprints are in-person events where our functions come together to plan roadmaps, share learnings, collaborate and work on initiatives&hellip; and simply have great fun!  The locations for these sprints vary but are usually between EMEA and AMER and they usually last 1 week, occasionally 2 depending on your involvement. For some roles i.e. Sales, Field, Alliances etc there will be a more regular requirement to travel. Please make sure that you ask about the specific requirements during your interview with the team. If you have any restrictions on travel please, please let us know.</p>
```